cmd/umeed/cmd/root.go

```diff
@@ -93,6 +95,45 @@ func initTendermintConfig() *tmcfg.Config {
 	return cfg
 }
 
+// initAppConfig helps to override default appConfig template and configs.
+// return "", nil if no custom configuration is required for the application.
+func initAppConfig() (string, interface{}) {
+	// WASMConfig defines configuration for the wasm module.
+	type WASMConfig struct {
```

Comment:
this shouldn't be done after we enable the `x/wasm` module?  

I am not against adding it now if you want...

Reply:
I added it for completness of this function,